std_msgs/Header header

bool having_risk
bool need_stop_now
bool high_risk
bool low_risk

int32 gear_theta_limit
int32 gear_r_limit
int32 gear_z_limit
